pinot-controller/src/main/java/org/apache/pinot/controller/tuner/RealTimeAutoIndexTuner.java

```diff
@@ -42,4 +42,10 @@ public TableConfig apply(PinotHelixResourceManager pinotHelixResourceManager,
     initialIndexingConfig.setNoDictionaryColumns(schema.getMetricNames());
     return tableConfig;
   }
+
+  @Override
+  public TableConfig apply(PinotHelixResourceManager pinotHelixResourceManager,
+      TableConfig tableConfig, Schema schema, Map<String, String> extraProperties, Map<String, String> httpHeaders) {
+    return apply(pinotHelixResourceManager, tableConfig, schema, extraProperties);
```

Comment:
are we ignoring the httpHeaders here?

Reply:
Yes its ignored for this case since there are no remote calls being made for this TableConfigTuner impl. Shall we just name the param `ignored` to make it clearer?